devtools/conda-envs/test_env.yaml

```diff
@@ -1,37 +1,19 @@
 name: openmmforcefields-test
-
 channels:
+  - conda-forge/label/openmm_rc
   - conda-forge
-
 dependencies:
-    # Base depends
   - python
   - pip
-
-    # Testing
   - pytest
   - pytest-cov
   - pytest-xdist
   - pytest-randomly
-
-    # Requirements for converted force field installer
-  - openmm >=7.6.0
-
+  - openmm ~=8.1.2rc1
   - openff-toolkit >=0.11
-
-    # Requirements for conversion tools
   - pyyaml
   - ambertools >=22,<24
   - lxml
   - networkx
-
-    # JSON caching of residue templates
   - tinydb
-
-    # Validating URLs
   - validators
-
-    #
-    # Espaloma requirements below
-    #
-  # espaloma >=0.3.1
```

Comment:
```
  error    libmamba Could not solve for environment specs
      The following packages are incompatible
      └─ espaloma >=0.3.1  is not installable because there are no viable options
         ├─ espaloma 0.3.1 would require
         │  └─ openmmforcefields >=0.11.2  with the potential options
         │     ├─ openmmforcefields [0.11.2|0.12.0] would require
         │     │  └─ ambertools >=20.0,<23 , which does not exist (perhaps a missing channel);
         │     └─ openmmforcefields 0.13.0 would require
         │        └─ espaloma >=0.3.2 , which can be installed;
         └─ espaloma 0.3.2 would require
            └─ dgl 1.1.2.* , which does not exist (perhaps a missing channel).
  critical libmamba Could not solve for environment specs

Reply:
https://github.com/openmm/openmmforcefields/issues/343